cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SubqueryExpressionsTest.scala

```diff
@@ -140,5 +144,107 @@ class SubqueryExpressionsTest extends DocumentingTest {
         }
       }
     }
+
+    section("COUNT subqueries", "count-subqueries") {
+      p("An count subquery can be used to find out how many times a specified pattern appears in the data.")
+      functionWithCypherStyleFormatting(
+        """COUNT {
+          #  [Pattern]
+          #  WHERE [Expression]
+          #}""".stripMargin('#'))
+      p("It is worth noting that the `MATCH` keyword can be omitted in subqueries and that the `WHERE` clause is optional.")
+
+      section("Simple count subquery", "count-subquery-simple-case") {
```

Comment:
```suggestion
      section("Simple `COUNT` subquery", "count-subquery-simple-case") {
```
... there are more cases like this one.